### DISCUSSION TITLE
Death reset

### DISCUSSION THREAD

**Discussion by AJ-Pullup** [Tictim/Paraglider#86] [Q&A]
Hello ! First I love you're mod the vibe that give to minecraft and it is shapped perfectly for my (private) server.

I just encounter difficulty to put something in place, I want player to go back to 20 health back.
I see you mad an attribute for stamina but not for the health and it don't seem to be affected via "minecraft:generic_maxHealth" and I look the easy config file you made for client and server but I didn't find clue about this so it's why I ask for some help.

Thank's anyway I wish you a great summer !

**Comment by Tictim** [comment]
The attributes given to players are managed by a separate internal data, so modifying attribute alone won't reset the player's heart.

Currently the only way to "reset" additional HP is to use the item Anti-Vessel, which gives back all consumed vessels - including Heart Containers and Stamina Vessels - as items. Addition of a command that can set the amount of vessels for player is planned.
